notebooks/logging-examples/logging-plots.ipynb

```diff
@@ -0,0 +1,318 @@
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>direction</th>\n",
+       "      <th>strength</th>\n",
+       "      <th>frequency</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>N</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>NNE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.6</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>NE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.5</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>ENE</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.4</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>E</td>\n",
+       "      <td>0-1</td>\n",
+       "      <td>0.4</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  direction strength  frequency\n",
+       "0         N      0-1        0.5\n",
+       "1       NNE      0-1        0.6\n",
+       "2        NE      0-1        0.5\n",
+       "3       ENE      0-1        0.4\n",
+       "4         E      0-1        0.4"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import plotly.express as px\n",
+    "from plotly import data\n",
+    "\n",
+    "df = data.wind()\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "31e007bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scatter_plot=px.scatter(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "#save our image for plotting\n",
+    "scatter_plot.write_image(\"saved-scatter-plot-for-logging.png\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "10688243",
+   "metadata": {},
+   "source": [
+    "![figure](saved-scatter-plot-for-logging.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "8544305b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bar_plot=px.bar(df, x=\"direction\", y=\"frequency\", color=\"strength\")\n",
+    "bar_plot_bytes=bar_plot.to_image(format=\"png\")"
```

Comment:
can you include some text in a markdown cell here and above explaining the difference between `to_image` and `write_image` and why were showing both?

Reply:
if the point of this `to_image` cell is to show that we can write to bytes as well as just saving to a file, I think showing a screen shot of the bar graph is a bit confusing since that isn't what the code is producing

Reply:
Yeah I was just showing that you can do it in memory or by writing a file